### DISCUSSION TITLE
I need some advice on trynna create a preset for a early to mid 1970s tv signal

### DISCUSSION THREAD

**Discussion by rekkol** [ntsc-rs/ntsc-rs#117] [Q&A]
Hey, so I'm trynna to create a early color tv effect preset from the late 60s to early to mid 70s any advice?
I tried creating it already in the App but for some reason it doesn't look quite right

It looks something like this:
![videoframe_16807](https://github.com/valadaptive/ntsc-rs/assets/172085024/5db2fb50-4409-41e8-95aa-0b43debd6fa6)

Notice the artifacts, (I blew it up to higher quality so you can see better)

Also I'm trynna create a early Black and white TV effect like this
![videoframe_3729](https://github.com/valadaptive/ntsc-rs/assets/172085024/198cc8a0-33fb-4077-9be9-e37c022d5776)
(Yea ik it says 1970 but it looks the same as early tv from the 50s and 60s)

any help? thanks

**Comment by dawnch0rus** [comment]
lots of luma smear. ringing. low pass on the chroma. check out demo videos of u-matic or 1" type c

**Comment by Degamisu** [comment]
I actually have a preset that mimics the B&W tapes of the era. If you need for reference, just ask!

**Reply by rekkol** [reply]
do u have the preset ?

**Reply by Degamisu** [reply]
> do u have the preset ?

Hi! sorry for the late reply. yes i do. here:

[BW70s.json](https://github.com/user-attachments/files/15848251/BW70s.json)
